app/src/main/java/org/fossasia/openevent/app/data/db/configuration/OrgaDatabase.java

```diff
@@ -18,23 +18,16 @@
 public class OrgaDatabase {
     public static final String NAME = "orga_database";
     // To be bumped after each schema change and migration addition
-    public static final int VERSION = 2;
+    public static final int VERSION = 4;
 
     // Migration version is less than or equal to Database version
     // Older version migrations are not to be removed
-    @Migration(version = 2, database = OrgaDatabase.class)
```

Comment:
The class is deleted. It is not possible. Besides, this migration will only delete the previous data on this version, so previous migrations won't matter